packages/calypso-e2e/src/lib/pages/gutenberg-editor-page.ts

```diff
@@ -336,6 +336,23 @@ export class GutenbergEditorPage {
 		await frame.fill( selectors.blockSearch, text );
 	}
 
+	/**
+	 * Verify that an entry with the expected name is shown in the
+	 * block inserter search result.
+	 *
+	 * @param {string} text Text to search.
+	 * @returns {boolean} True if an entry with exact name match is found. False otherwise.
+	 */
+	async validateBlockShownInResult( text: string ): Promise< boolean > {
```

Comment:
Previously `GutenbergEditorComponent` would implicitly verify that expected block(s) were being shown in the block inserter panel by attempting to click on a block with the matching name.